DOC-841: Add docs for new PowerPaste import settings functionality (#2022)
tinymce/tinymce-docs: release-notes/release-notes59.md

For information on the <<Premium Plugin Name>> plugin, see: [<<Premium Plugin Name>> plugin]({{site.baseurl}}/plugins/<<Premium Plugin Name>>/).

### PowerPaste 5.6.0

The {{site.productname}} 5.9 release includes an accompanying release of the **PowerPaste** premium plugin.

> **PowerPaste** 5.6.0 introduces the following enhancements:

Author: I think 5.6.0 is right? Got it from the current 5-dev PowerPaste version.

> - Asynchronous functions are now possible for the `powerpaste_word_import`, `powerpaste_googledocs_import` and `powerpaste_html_import` options. This allows the paste mode to be dynamically chosen on each paste event. It is designed to be used to replicate the `prompt` dialog with custom, non-TinyMCE UI.

Reviewer: ```suggestion
- Asynchronous functions are now possible for the `powerpaste_word_import`, `powerpaste_googledocs_import`, and `powerpaste_html_import` options. This allows the paste mode to be dynamically chosen on each paste event. It is designed to be used to replicate the `prompt` dialog with custom, non-TinyMCE UI.
```

> - Asynchronous functions are now possible for the `powerpaste_word_import`, `powerpaste_googledocs_import`, and `powerpaste_html_import` options. This allows the paste mode to be dynamically chosen on each paste event. It is designed to be used to replicate the `prompt` dialog with custom, non-TinyMCE UI.

Author: ```suggestion
- The `powerpaste_word_import`, `powerpaste_googledocs_import`, and `powerpaste_html_import` options now also accept asynchronous functions. This allows the paste mode to be dynamically set each time a user pastes relevant content. It is designed to be used, for example, to replicate the `prompt` dialog with a custom dialog.
```

> - Asynchronous callback functionality added to the `powerpaste_word_import`, `powerpaste_googledocs_import` and `powerpaste_html_import` options. This allows the integrator to dynamically choose the paste mode on each paste event. It is designed to be used to replicate the `prompt` dialog with custom, non-TinyMCE UI.

Reviewer 2: ```suggestion
- Asynchronous callback functionality added to the `powerpaste_word_import`, `powerpaste_googledocs_import` and `powerpaste_html_import` options. This allows the paste mode to be dynamically chosen on each paste event. It is designed to be used to replicate the `prompt` dialog with custom, non-TinyMCE UI.
```
I don't think I've ever seen us say `the integrator` especially when they are the target audience.

Author: Me neither, but I couldn't think of anything better yesterday. Thanks!

Author: `now possible` -> `now valid values` or something? That part reads oddly to me...

> - The `powerpaste_word_import`, `powerpaste_googledocs_import`, and `powerpaste_html_import` options now also accept asynchronous functions. This allows the paste mode to be dynamically set each time a user pastes relevant content. It is designed to be used, for example, to replicate the `prompt` dialog with a custom dialog.

Reviewer 2: ```suggestion
- The `powerpaste_word_import`, `powerpaste_googledocs_import`, and `powerpaste_html_import` options now accept asynchronous functions. This allows the paste mode to be dynamically set each time a user pastes relevant content. It is designed to be used, for example, to replicate the `prompt` dialog with a custom dialog.
```

> For information on the PowerPaste plugin, see: [PowerPaste plugin]({{site.baseurl}}/plugins/premium/PowerPaste/).

Author: ```suggestion
For information on the PowerPaste plugin, see: [PowerPaste plugin]({{site.baseurl}}/plugins/premium/powerpaste/).
```
- [ ] todo
